docs/hub/tensorboard.md

```diff
@@ -4,7 +4,7 @@ TensorBoard provides tooling for tracking and visualizing metrics as well as vis
 
 ## Exploring TensorBoard models on the Hub
 
-Over 6,000 repositories have TensorBoard traces on the Hub. You can find them by filtering at the left of the [models page](https://huggingface.co/models?filter=tensorboard). As an example, if you go to the [pyannote/embedding](https://huggingface.co/aubmindlab/bert-base-arabertv02) repository, there is a **Metrics** tab. If you select it, you'll view a TensorBoard instance.
+Over 6,000 repositories have TensorBoard traces on the Hub. You can find them by filtering at the left of the [models page](https://huggingface.co/models?filter=tensorboard). As an example, if you go to the [aubmindlab/bert-base-arabertv02](https://huggingface.co/aubmindlab/bert-base-arabertv02) repository, there is a **Metrics** tab. If you select it, you'll view a TensorBoard instance.
```

Comment:
Btw the 6k number is a bit outdated as we now have [52k+ public models](https://huggingface.co/models?library=tensorboard) with tensorboard traces

```suggestion
Over 50k repositories have TensorBoard traces on the Hub. You can find them by filtering at the left of the [models page](https://huggingface.co/models?filter=tensorboard). As an example, if you go to the [aubmindlab/bert-base-arabertv02](https://huggingface.co/aubmindlab/bert-base-arabertv02) repository, there is a **Metrics** tab. If you select it, you'll view a TensorBoard instance.
```